pkg/aws/utils.go

```diff
@@ -151,13 +151,13 @@ func GetAWSClientForUserRegion(reporter *rprtr.Object, logger *logrus.Logger) Cl
 // Validations will validate if CF stack/users exist
 func CheckStackReadyForCreateCluster(reporter *rprtr.Object, logger *logrus.Logger) {
```

Comment:
I think we should rename this function, or remove it. We should be able to call `awsClient.CheckAdminUserExists` directly from `cmd/create/cluster/cmd.go` and track rprtr/logs there